src/sdl2_compat.c

```diff
@@ -1880,6 +1953,9 @@ Event2to3(const SDL2_Event *event2, SDL_Event *event3)
             }
         }
         break;
+    case SDL_EVENT_SENSOR_UPDATE:
+        event3->sensor.which = SensorID2to3(event2->sensor.which);
+        break;
```

Comment:
need `SDL_US_TO_NS()` for the timestamp here?

Reply:
Yep!